### PR TITLE
add new arguments

### PR DESCRIPTION
**Pull request by cdonnay**
Add a `rank_cols` argument to `load_csv` so that the user can specify which columns of the csv correspond to rankings. Helpful, for example, with the 2013 Minneapolis data file on MGGG/chicago which has extraneous columns.

Also removed `quota` parameter from `Election.get_threshold` method since `quota` should be an attribute of the Election.

Also, this is my first pull request; Moon has asked me to tag y'all going forward. Let me know if you don't wish to be tagged or if I made an error in my request. Thanks!
@jamesturk @drdeford @jgibson517 @ziglaser @jennjwang

### CODE REVIEW

**Comment by jamesturk** [comment]
Hi @cdonnay,

A couple of notes:

1) You'll want to add .idea to .gitignore, and remove the .idea files, those are only relevant to you/PyCharm and shouldn't be part of this repo.
2) You're adding a required parameter, but it should probably be optional since it was already working without that.  This is also why the tests are failing.  (A new test should probably be added to accept this optional parameter.)

I'll let someone else comment on the self.quota issue as I haven't kept up with the API in that level of detail.

**Comment by cdonnay** [comment]
Thanks, @jamesturk ! I added .idea to .gitignore, and gave the `rank_cols` a default argument. 

I think it's worth discussing whether or not you actually want the `rank_cols` parameter to have a default argument. My argument for not including a default is that we don't want to force preprocessing of the csv file onto the user, and we want the user to think intentionally about loading in their data. Happy to hear other ideas about why we might want a default.

**Comment by jennjwang** [comment]
Hi @cdonnay thanks for helping with Votekit! I agree with James on making rank_cols an optional argument - I think the user should have an idea of what their data looks like before loading it, but it would be annoying for them to specify the rank_cols every time if their csv file is already clean without extraneous columns. A default would save them this trouble.

**Comment by cdonnay** [comment]
Awesome, sounds good to me.

File: src/votekit/elections/election_types.py [hunk 1]
```diff
@@ -61,20 +61,18 @@ def __init__(
         self.transfer = transfer
         self.seats = seats
         self.tiebreak = tiebreak
-        self.threshold = self.get_threshold(quota)
+        self.quota = quota.lower()
```
Comment (jgibson517) [inline src/votekit/elections/election_types.py:64]: 👍 agree with this change

File: src/votekit/cvr_loaders.py [hunk 2]
```diff
@@ -55,6 +59,12 @@ def load_csv(
     if id_col is not None and not df.iloc[:, id_col].is_unique:
         raise DataError(f"Duplicate value(s) in column at index {id_col}")
 
+    if rank_cols:
```
Comment (jgibson517) [inline src/votekit/cvr_loaders.py:62]: 👍

File: .gitignore [hunk 3]
```diff
@@ -5,4 +5,4 @@ htmlcov/
 .DS_Store
 dist/
 .ipynb_checkpoints
-
+.idea
```
Comment (jgibson517) [inline .gitignore:8]: you may need to delete the .idea and inspectionProfiles files from the version of your repo on github, but then the .gitignore should handle them in the future!

Comment (cdonnay) [reply .gitignore:8]: Okay, I think I just deleted those files.

**Pull request merged**